src/app/core/models/price/price.pipe.spec.ts

```diff
@@ -30,11 +29,8 @@ describe('Price Pipe', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [DummyComponent, PricePipe],
-      imports: [
-        TranslateModule.forRoot(),
-        ngrxTesting({ reducers: { configuration: configurationReducer, user: userReducer } }),
-      ],
-      providers: [PricePipe],
+      imports: [TranslateModule.forRoot()],
+      providers: [PricePipe, provideMockStore()],
```

Comment:
why is provideMockStore used here (as an example, there are a lot of similar other occurances), is there a rule in which cases mocking should be used instead of importing?

Reply:
Documentation will follow :slightly_smiling_face: